.github/workflows/ci.yml

```
@@ -5,10 +5,16 @@ permissions:
   pull-requests: write
 
 on:
+  # release/** is covered explicitly rather than incidentally. A release branch is only built
+  # today because an open release PR happens to point at main, which makes its head a pull_request
+  # head — so coverage disappears the moment that PR merges or closes.
   push:
-    branches: [ main, development]
+    branches: [ main, development, 'release/**' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)